pagopa/io-app
# fix(Firma con IO): [SFEQS-1568] Clear FCI state when user reset pin during signing flow

## Short description
This PR fix an issue related to the reset pin request by the user during a signing flow. When user reset the pin the state should be cleared to avoid dirty data before start a new signing flow.

## List of changes proposed in this pull request
- Update FCI saga to take IDENTIFICATION_PIN_RESET

## How to test
Open a tool like reacttotron and watch the app state for fci. Using io-dev-api-server try to start a signing flow. On the identification pin screen try to reset the pin. The FCI state should be cleared.
